packages/core/src/indexing/local-index-api.ts

```diff
@@ -46,7 +46,7 @@ async function _getIndexModelArgs(
         relations: content.relations,
       }
     }
-    const indices = (await databaseIndexApi?.getFieldsIndex(modelStreamId)) ?? req.indices
+    const indices = req.indices ?? (await databaseIndexApi?.getFieldsIndex(modelStreamId))
```

Comment:
Why do we need to check the database for existing indices here?  I think I asked about this in an earlier review round also and never got a response.

Reply:
This is for nodes that come back up with id's pinned for indexing. We can then validate that tables exist in the proper state or need to be migrated.

Reply:
Sorry, I don't understand this answer.  What do you mean "with id's pinned for indexing"?